### ISSUE TITLE
License

### ISSUE THREAD

**Issue by vmario89** [Hyvok/Inkscape-rotary-encoder-disk-generator#7]
Hi dude,
i love this plugin. You did not apply some license to this repository.

May i use it for some remix? Most InkScape plugins are licensed by GPL v2/v3 or MIT License. Would be happy if you add some license to this repo.

regards, Mario

**Comment by Dragonrun1** [comment]
Sorry I didn't reply soon the holidays and new job have been keeping me busy. I have no problem with you using it under any open source license but since this is just a branch from the upstream project to do some bug fixing you might need to contact him about it. My impression was he really wasn't that interested in the project anymore but that could have changed.

**Comment by Hyvok** [comment]
Hello! Thank you for the interest in my project :)

I've added an MIT license for the plugin, so have fun with it! ea16415f7264d43108d4a92051c9601aca92218a

I haven't done anything with the plugin for years, but actually just yesterday I was updating it for Inkscape v1.0 which has changed the inkex API quite a bit. Still have some issue with it but that will be hopefully incoming soon.

**Comment by Dragonrun1** [comment]
I've also now updated my fork with the license as well. Thanks Kalle for clarifying it for us.

    On Tuesday, February 18, 2020, 10:35:17 PM PST, Kalle Hyvönen <notifications@github.com> wrote:  
 
 
Hello! Thank you for the interest in my project :)

I've added an MIT license for the plugin, so have fun with it! ea16415

I haven't done anything with the plugin for years, but actually just yesterday I was updating it for Inkscape v1.0 which has changed the inkex API quite a bit. Still have some issue with it but that will be hopefully incoming soon.
